api/types.go

```diff
@@ -733,3 +733,13 @@ type SsmCommandRequest struct {
 	Parameters     map[string][]*string `json:"parameters"`
 	TimeoutSeconds *int64               `json:"timeout"`
 }
+
+type Ec2VolumeUpdateTagRequest struct {
+	Tags map[string]string
```

Comment:
You can get rid of this type and move the Tags into the other one.

Reply:
Done.